radixdlt-core/radixdlt/build.gradle

```diff
@@ -149,6 +149,13 @@ dependencies {
     implementation 'com.stijndewitt.undertow.cors:undertow-cors-filter'
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
 
+    implementation 'io.vavr:vavr:1.0.0-alpha-3'
```

Comment:
this is for `Either`

Reply:
Not sure that we actually need `Either`. In most cases we need something like 'Result' rather than `Either`. And such a class is rather simple, so we might prefer to implement them by ourselves rather than add more dependencies.

Reply:
union types are very common, from my experience it's quite useful to have one around